From 2a6ce8b5214636d7c98cf973a0f8d524c54426d1 Mon Sep 17 00:00:00 2001
From: Bernd Naumann <bernd@kr217.de>
Date: Fri, 13 Jun 2014 15:34:08 +0200
Subject: [PATCH] [ntpd] support for DCF77

[ntpd] support for DCF77

enable support local clock and DCF77 - enable full build

Signed-off-by: Bernd Naumann <bernd@kr217.de>
---
 net/ntpd/Makefile |    8 +-------
 1 files changed, 1 insertions(+), 7 deletions(-)

diff --git a/net/ntpd/Makefile b/net/ntpd/Makefile
index 881887c..d0daf1d 100644
--- a/net/ntpd/Makefile
+++ b/net/ntpd/Makefile
@@ -123,14 +123,8 @@ CONFIGURE_VARS += \
 	ac_cv_header_dns_sd_h=no
 
 CONFIGURE_ARGS += \
-	--disable-all-clocks \
-	--disable-parse-clocks \
-	--without-ntpsnmpd \
-	--without-lineeditlibs \
-	--enable-NMEA \
 	--enable-LOCAL-CLOCK \
-	--enable-SHM \
-	--enable-linuxcaps
+	--enable-RAWDCF
 
 ifeq ($(BUILD_VARIANT),ssl)
   CONFIGURE_ARGS += \
-- 
1.7.2.5

